tests/utils/accounts.py

```diff
@@ -0,0 +1,12 @@
+from starkware.starknet.testing.starknet import StarknetContract
+
+
+async def fund_evm_address(
```

Comment:
make it a fixture to have `kakarot` and `eth` already existing for the "end user" (the dev writing a test)

Reply:
done.